python/pickle/PyROOT_readtests.py

```diff
@@ -107,12 +107,27 @@ def __doftest( self, d ):
    def test5ReadCustomTypes( self ):
       """Test reading PyROOT custom types"""
 
-      o = pickle.load( self.in1 )
-      self.assertEqual(o, [123,123.123] )
+      self.exp_pyroot = os.environ.get('EXP_PYROOT') == 'True'
```

Comment:
If you only use `exp_pyroot` in this function, you do not need to bind it to self, just use a local variable.